### ISSUE TITLE
Confluent Replicator Demo Issue: Reporting error "Waiting up to 180 seconds for Connect to start"

### ISSUE THREAD

**Issue by GEETAREDDYK** [confluentinc/examples#1019]
Hello,

Following the steps to run the demo on Confluent replicator. It is stuck while launching the Connect cluster with the error shown below in the output. Appreciate any input. 

**Link**: https://docs.confluent.io/platform/current/multi-dc-deployments/replicator/replicator-docker-tutorial.html#environment

**Output:**
Creating broker-dc1    ... done
Creating schema-registry-dc1 ... done
Creating schema-registry-dc2 ... done
Creating datagen-dc1-topic1  ... done
Creating datagen-dc1-topic2  ... done
Creating connect-dc2         ... done
Creating datagen-dc2-topic1  ... done
Creating connect-dc1         ... done
Creating control-center      ... done
Waiting up to 180 seconds for Connect to start
.......................
Waiting up to 180 seconds for Connect to start
....................................ERROR: Failed after 180 seconds. Please troubleshoot and run again.

**Comment by GEETAREDDYK** [comment]
This is how my docker containers status looks like. Not sure some of the critical services are down. Might be this is the reason (brokers are down as shown in below image), Connector instance could not able to up and running. As this is demo scene, are there an issues with the services supported from origin cluster?

![image](https://user-images.githubusercontent.com/58374511/130376855-b908c14f-1524-4c22-bf70-a54e3527c2e5.png)

**Comment by ybyzek** [comment]
@GEETAREDDYK looks like the brokers are down.  Please check for error messages in the broker logs (e.g. `docker-compose logs broker-dc1`).

**Comment by GEETAREDDYK** [comment]
@ybyzek , yes they are down. One or other service is keep going down. Even if I make them up, a specific service will stay active for a few mins and again going down. Am I missing anything?

**Comment by GEETAREDDYK** [comment]
Just now I have tried one more time. Initially the services are up, but just 
![image](https://user-images.githubusercontent.com/58374511/130453705-295cb587-1934-48d4-9698-fa956ee2b997.png)

**Comment by GEETAREDDYK** [comment]
My apologies. Accidentally closed the issue here. Reopened again.
I am seeing below INFO and TRACE details in the log. Seems some configuration issues. 
![Uploading image.png…]()
broker-dc1             |  (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'replication.factor' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'confluent.metrics.reporter.bootstrap.servers' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'confluent.support.customer.id' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'confluent.metrics.reporter.topic.replicas' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'min.insync.replicas' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'confluent.metrics.enable' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] WARN The configuration 'jmx.port' was supplied but isn't a known config. (org.apache.kafka.clients.admin.AdminClientConfig)
broker-dc1             | [2021-08-23 13:15:46,216] INFO Kafka version: 6.2.0-ce (org.apache.kafka.common.utils.AppInfoParser)
broker-dc1             | [2021-08-23 13:15:46,216] INFO Kafka commitId: eba41251e36e4cfb (org.apache.kafka.common.utils.AppInfoParser)
broker-dc1             | [2021-08-23 13:15:46,216] INFO Kafka startTimeMs: 1629724546216 (org.apache.kafka.common.utils.AppInfoParser)
broker-dc1             | [2021-08-23 13:15:46,369] INFO Creating topic _confluent-license with configuration {cleanup.policy=compact, min.insync.replicas=1} and initial partition assignment HashMap(0 -> ReplicaAssignment(replicas=1, addingReplicas=, removingReplicas=, observers=, targetObservers=None)) (kafka.zk.AdminZkClient)
broker-dc1             | [2021-08-23 13:15:46,718] INFO [Controller id=1] New topics: [Set(_confluent-license)], deleted topics: [HashSet()], new partition replica assignment [Set(TopicIdReplicaAssignment(_confluent-license,Some(ybc8yTqjRqiBbj8YDl1L1A),None,Map(_confluent-license-0 -> ReplicaAssignment(replicas=1, addingReplicas=, removingReplicas=, observers=, targetObservers=None)),None))] (kafka.controller.KafkaController)
broker-dc1             | [2021-08-23 13:15:46,720] INFO [Controller id=1] New partition creation callback for _confluent-license-0 (kafka.controller.KafkaController)
broker-dc1             | [2021-08-23 13:15:46,747] INFO [Controller id=1 epoch=2] Changed partition _confluent-license-0 state from NonExistentPartition to NewPartition with assigned replicas 1 (state.change.logger)
broker-dc1             | [2021-08-23 13:15:46,749] TRACE [Controller id=1 epoch=2] Changed state of replica 1 for partition _confluent-license-0 from NonExistentReplica to NewReplica (state.change.logger)
broker-dc1             | [2021-08-23 13:15:46,974] INFO [Controller id=1 epoch=2] Changed partition _confluent-license-0 from NewPartition to OnlinePartition with state LeaderAndIsr(leader=1, leaderEpoch=0, isUncleanLeader=false, isr=List(1), zkVersion=0) (state.change.logger)
broker-dc1             | [2021-08-23 13:15:46,976] TRACE [Controller id=1 epoch=2] Sending become-leader LeaderAndIsr request LeaderAndIsrPartitionState(topicName='_confluent-license', topicId=AAAAAAAAAAAAAAAAAAAAAA, clusterLinkId=null, clusterLinkTopicState=null, linkedLeaderEpoch=-1, partitionIndex=0, controllerEpoch=2, leader=1, confluentIsUncleanLeader=false, leaderEpoch=0, isr=[1], zkVersion=0, replicas=[1], observers=[], addingReplicas=[], removingReplicas=[], isNew=true) to broker 1 for partition _confluent-license-0 (state.change.logger)
broker-dc1             | [2021-08-23 13:15:46,976] INFO [Controller id=1 epoch=2] Sending LeaderAndIsr request to broker 1 with 1 become-leader and 0 become-follower partitions (state.change.logger)
broker-dc1             | [2021-08-23 13:15:46,976] TRACE [Controller id=1 epoch=2] Changed state of replica 1 for partition _confluent-license-0 from NewReplica to OnlineReplica (state.change.logger)
broker-dc1             | [2021-08-23 13:15:47,005] INFO [Broker id=1] Handling LeaderAndIsr request correlationId 3 from controller 1 for 1 partitions (state.change.logger)
